spec/support/integration_setup.rb

```diff
@@ -31,6 +32,14 @@ def prepare_app
     end
   end
 
+  def use_local_gris
+    IO.write(
+      "#{app_name}/Gemfile", File.open("#{app_name}/Gemfile") do |f|
+        f.read.gsub(/gem 'gris'/, "gem 'gris', path: '../'")
+      end
+    )
+  end
```

Comment:
Rewrites the gris gem declaration to point to the local version of the codebase rather than a publicly distributed Gris.